src/lib.rs

```diff
@@ -21,15 +22,24 @@ use std::{
 };
 
 use clap::Parser;
-pub use config::{Config, ConfigBuilder};
-use context::{PendingFix, QueryMatchContext};
+pub use cli::bootstrap_cli;
+pub use config::{Args, Config, ConfigBuilder};
+use context::PendingFix;
+pub use context::QueryMatchContext;
+pub use proc_macros::{builder_args, rule, rule_tests};
```

Comment:
Re-export the "publicly useful" proc-macros (from the crate root)

`builder_args` needs to be here because eg the `violation!()` declarative macro delegates to it (so would emit eg `tree_sitter_lint::builder_args!(...)`)